resources/views/omnomcom/store/show.blade.php

```diff
@@ -721,9 +721,18 @@
 
     <div id="buttons">
 
-        <span id="purchase" class="button inactive">Complete order</span>
+        <span id="purchase" class="button inactive">
+            <i class="fas fa-cookie-bite mr-2"></i> Complete order
+        </span>
         @if($store->cash_allowed)
-            <span id="purchase-cash-initiate" class="button inactive">Complete with cash</span>
+            <span id="purchase-cash-initiate" class="button inactive">
+                <i class="fas fa-coins mr-2"></i> Complete with cash
+            </span>
+        @endif
+        @if($store->bank_card_allowed)
+            <span id="purchase-bank-card-initiate" class="button inactive">
+                <i class="fas fa-credit-card mr-2"></i> Complete with PIN
```

Comment:
PIN might suggest that the payment is completed with a PIN code instead of a bank card.